knotx-core/knotx-fragment-assembler/src/main/java/com/cognifide/knotx/knot/assembler/AssemblyStrategy.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Knot.x - Reactive microservice assembler - Fragment Assembler
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.knot.assembler;
+
+import com.cognifide.knotx.fragments.Fragment;
+import com.cognifide.knotx.fragments.FragmentConstants;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+
+public enum AssemblyStrategy {
```

Comment:
As mentioned in the comments agains wiki, I'd call this class `UnprocessedStrategy` or something like this to clearly say this is the strategy how to handle unprocessed snippets.